src/bloaty.cc

```diff
@@ -182,15 +181,41 @@ std::string ItaniumDemangle(string_view symbol, DataSource source) {
     if (absl::debugging_internal::Demangle(demangle_from.data(), demangled,
```

Comment:
We're somewhat interested in moving off of absl's `Demangle`, could we replace that with demumble as well? 

This could be a follow up issue / discussion, just wanted to highlight it while we're here.